Sources/ShortcutFoundation/Core/Device/Biometrics/DeviceAuthentication.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  DeviceAuthentication.swift
+//  
+//
+//  Created by Swathi on 2022-02-02.
+//
```

Comment:
@swathikumar-shortcut could you adjust the file header?

Reply:
@gabrielPeart : Updated. Please verify